packages/devtools/tests/launcher.test.js

```diff
@@ -151,14 +171,33 @@ test('throws if browser is unknown', async () => {
     }
 })
 
-test('launch Firefox without Puppeteer default args', async () => {
+test('launch Firefox Nightly without Puppeteer default args', async () => {
     await launch({
         browserName: 'firefox',
+        'moz:firefoxOptions': {
+            binary: '/foo/firefox-nightly',
+            headless: true
+        },
         ignoreDefaultArgs: true
     })
     expect(puppeteer.launch.mock.calls).toMatchSnapshot()
 })
 
+test('launch Firefox without Puppeteer default args', async () => {
+    try {
+        await launch({
+            browserName: 'firefox',
+            'moz:firefoxOptions': {
+                binary: '/foo/firefox',
+                headless: true
+            },
+            ignoreDefaultArgs: true
+        })
+    } catch (err) {
+        expect(err.message).toContain('Only Nightly release channel is supported')
+    }
```

Comment:
This test would also pass if `launch` would not throw an error here. You can either use `expect.assertions(1)` or what I prefer:

```suggestion
    const err = await launch({
        browserName: 'firefox',
        'moz:firefoxOptions': {
            binary: '/foo/firefox',
            headless: true
        },
        ignoreDefaultArgs: true
    }).catch(err => err)
    expect(err.message).toContain('Only Nightly release channel is supported')
```

Reply:
Handled it with `expect.assertions(1)`